lib/zeus/api_client.rb

```diff
@@ -25,14 +25,19 @@ class APIClient
 
     # constructor for Zeus::APIClient
     # @param       [Hash] opts the options to create z Zeus::APIClient instance
-    # @option opts [String] :access_token The tokens for Zeus Service
+    # @option opts [String] :token The tokens (user token or access token) for Zeus Service
```

Comment:
For user, maybe access_token is more make sense?
(Actually I call usertoken as  'access_token' in everywhere.)
Just preference.